index.html

```diff
@@ -2,98 +2,111 @@
-            </div>
-        </div>
+<div class="hero-body rnw-hero">
+  <div class="container has-text-centered">
+    <div class="columns is-vcentered">
+      <div class="column is-5">
+        <figure class="image is-4by3">
+          <img src="/images/rubymeet.jpeg" alt="Ruby Meetup">
+        </figure>
+      </div>
+      <div class="column is-6 is-offset-1">
+        <h1 class="title is-2">
+          Welcome, Fam!
+        </h1>
+        <h2 class="subtitle is-4">
+          {{ site.description }}
+        </h2>
+        <br>
+        <p class="has-text-centered">
+          <a href="https://www.ruby-lang.org/en/" class="button is-danger is-rnw-btn">
+            Get Started With Ruby
+          </a>
+        </p>
+      </div>
     </div>
-    <section>
-        <div class="sandbox pdt-50">
-            <div class="tile is-ancestor">
-                <div class="tile is-parent is-shady">
-                    <article class="tile is-child notification is-white">
-                        <p class="title">Join RubyNepal</p>
-                        <p class="subtitle">
-                            RubyNepal is a great way to meet and collaborate with other Ruby/Rails developers in Nepal.
-                            Joining Ruby Nepal is free.
-                        </p>
-                    </article>
-                </div>
-                <div class="tile is-parent is-shady">
-                    <article class="tile is-child notification is-white">
-                        <p class="title">Meetup Group</p>
-                        <p class="subtitle">
-                            We meet every 2nd Sunday of the month. <br>
-                            Nepal Ruby and Rails Users Group
-                        </p>
-                    </article>
-                </div>
-                <div class="tile is-parent is-shady">
-                    <article class="tile is-child notification is-white">
-                        <p class="title">Slack Chat</p>
-                        <div class="content">
-                            <p>Get your invite to join RubyNepal Slack channel</p>
-                        </div>
-                    </article>
-                </div>
-            </div>
+  </div>
+  <section>
+    <div class="sandbox pdt-50">
+      <div class="tile is-ancestor">
+        <div class="tile is-parent is-shady">
+          <article class="tile is-child notification is-white">
+            <p class="title">Join RubyNepal</p>
+            <p class="subtitle">
+              RubyNepal is a great way to meet and collaborate with other Ruby/Rails developers in Nepal.
+              Joining Ruby Nepal is free.
+            </p>
+          </article>
+        </div>
+        <div class="tile is-parent is-shady">
+          <article class="tile is-child notification is-white">
+            <p class="title">Meetup Group</p>
+            <p class="subtitle">
+              We meet every 2nd Sunday of the month. <br>
```

Comment:
This might not be relevant in the present scenario. So we might need to remove this and move forewared. @zoras dai

Reply:
This is true now as well. Let's keep this here as a reminder and motivation to ourselves.